docs/internal-documentation/maintenance/Instructions-for-translators.md

```diff
@@ -0,0 +1,401 @@
+  only commit the "xx.po" file.
+  ```
+  git commit -m 'Write a description of the change' xx.po
+  ```
+
+* There could be other files changed or added that should not be included.
+  Run the status command to see them.
+  ```
+  git status
+  ```
+
+* Other changed files could be reset by a `git checkout`. This could also
+  be done before creating the commit.
+  ```
+  git checkout FILENAME
+  ```
+
+* Added files not needed can just be removed. This could also be done
+  before the commit.
+  ```
+  rm FILENAME
+  ```
+
+* Now push the local branch you created to your fork at Github.
+  "translation-update" is name of the branch you created above and
+  have committed the updates to. Use your Github user name instead of
+  "xxxx".
+  ```
+  git push -u xxxx translation-update
+  ```
+
+* Go to your fork collection at Github, https://github.com/xxxx/ using your
+  Github user name instead of "xxxx". There you can select the fork for this
+  translation.
+
+* Select to create a new pull request. Using Zonemaster-Engine as an example,
+  the base repository should be *zonemaster/zonemaster-engine* and the base
+  branch should be *develop* (not *master*). The "head" should be your fork and
+  "compare" the same branch as you created above and pushed to your fork,
+  "translation-update".
+
+* Inspect what Github says that will change by the pull request. It should
+  only be the *PO* file that you have updated and nothing else. If additional
+  files are listed, please correct or request for help.
+  
+  * If you by mistake have not started with an updated version of the develop
+    branch you can get extra files listed.
+
+* If you want to correct something now before the pull request has been created,
+  do that in your local clone and do a new commit and push that to your fork.
+
+* Press "create pull request", write a nice description and press "create"
+  again.
+
+* If you go back to your own computer and just keep the clone as it is, you
+  can easily update the pull request if needed with more changes to the same
+  *PO* file. When the pull request has been merged by the Zonemaster work group,
+  you can delete the local clone and on your Github fork you can remove the
+  branch. Or keep them for next time.
+
+
+## Updates to msgids
+
+The *msgid* cannot be updated in the translation process. If you find an *msgid*
+that you feel is inferior then you can provide that information to the project.
+The preferred way is to create an issue in the relevant repository.
+
+* Message *(msgid)* in Zonemaster-Engine: [issue][new issue zonemaster-engine]
+* Message *(msgid)* in Zonemaster-CLI: [issue][new issue zonemaster-cli]
+* Message *(msgid)* in Zonemaster-Backend: [issue][new issue zonemaster-backend]
+
+If the message is in Zonemaster-Engine then include the message tag found in the
+header, e.g. "BASIC:NO_PARENT", in the issue decription. Also include the *msgid*
+as it is now and a suggestion for new language.
```

Comment:
Yes, I agree. But in this document we use the word "language" a lot. In every single case except this one we use it to mean things like "Swedish" or "French". In this particular case there is a perfectly good synonym (i.e. wording) that we could use to distinguish the meanings from each other. I do not think it's strictly necessary, but I feel it would be rather nice to the reader.